Cargo.toml

```diff
@@ -66,6 +66,7 @@ members = [
     "utils/litemap",
     "utils/pattern",
     "utils/resb",
+    "utils/system_locale",
```

Comment:
This name suggests that the crate only retrieves "system" "locale". My understanding is that your crate aims to retrieve environment information from the runtime. I'd use a term like "environment", "preferences" etc.

My concern about the "system" is that it indicates that what you get is "OS Locale" - but you may not. Increasingly OSes sandbox applications, so OS may be in English, but the application may only see that the language of its own environment is French.
My concern about "locale" is that it indicates that all you get is a Unicode Locale - while what you actually get are things beyond that. For example, short date pattern is not part of a Locale but should be retrievable by this crate.

Maybe "utils/env_preferences"?